internal/desktop_browser.py

```diff
@@ -359,6 +359,8 @@ def stop(self, job, task):
 
     def wait_for_idle(self, wait_time = 30):
         """Wait for no more than 50% CPU utilization for 400ms"""
+        if self.options.noidle:
```

Comment:
I noticed that the --noidle flag was not checked inside the desktop_browser.
Here is the output from each run.
Before Fix: https://pastebin.com/KXT6QUtn
After Fix: https://pastebin.com/B8CK7Mgb
![--noidleNotworking](https://user-images.githubusercontent.com/33207323/165160638-02c89927-fb15-4545-bf2f-5c222fd11adb.png)
![after_noidle_fix](https://user-images.githubusercontent.com/33207323/165160650-e80ce841-04d9-4b17-a295-e0f074862fb8.png)